api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1877,7 +1879,8 @@ public void copyObject(String bucketName, String objectName, String destBucketNa
         NoResponseException, ErrorResponseException, InternalException, IOException, XmlPullParserException,
         InvalidArgumentException {
 
-    copyObject(bucketName, objectName, destBucketName, null, copyConditions);
+    copyObject(bucketName, objectName, destBucketName, null, copyConditions,
+               null);
```

Comment:
This is not a cleanup @balamurugana it is reflecting the change in function signature.